stages/eib_manifest

```diff
@@ -23,4 +23,4 @@ run_hooks manifest
 
 # Assemble all the json fragments to one file
 mkdir -p "${EIB_OUTDIR}"
-"${EIB_HELPERSDIR}"/assemble-manifest > "$(eib_outfile manifest.json)"
+#"${EIB_HELPERSDIR}"/assemble-manifest > "$(eib_outfile manifest.json)"
```

Comment:
The image server is dependent on this file, so you still need to generate it.

Reply:
Hmm, ok. I was hoping we could remove it until https://github.com/endlessm/eos-build-meta/issues/76 is ready. But I could get it working again in this branch, so that we don't hit errors when uploading to the image server.

Reply:
Actually, I'm pretty sure the image server will survive if the file isn't included, but it's certainly desired. Are there errors from doing this or are you just worried the data will be wrong?